apps/snowflake/data/index.html

```diff
@@ -0,0 +1,130 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Tow Snowflake Proxy</title>
```

Comment:
There is a typo here. This should be 'Tor'

Reply:
Oops, good catch!